### ISSUE TITLE
When generating obfuscated DLLs, echo the obfuscated 'RunSliver' function to the sliver operator

### ISSUE THREAD

**Issue by gettheguru** [BishopFox/sliver#173]
**Is your feature request related to a problem? Please describe.**
When carrying out symbol obfuscation of the obfuscated Sliver shared library (DLL), the Run function (e.g. RunSliver) is also obfuscated. It would be useful to have the obfuscated function name informed to the operator when generating.

**Describe the solution you'd like**
```
sliver > generate --mtls 1.2.3.4 -r shared                                                                                                           
                                                                                                                                                             
[*] Generating new windows/amd64 Sliver binary                                                                                                               
[*] Symbol obfuscation is enabled.                                                                                                                           
[*] This process can take awhile, and consumes significant amounts of CPU/Memory                                                                             
[*] Build completed in 00:07:12 
[*] Exported RunSliver function is sjjjjjjsssssddhhhhhfff. Execute with 'rundll32.exe SLIVER_SLIVERY.dll,sjjjjjjsssssddhhhhhfff
```

**Describe alternatives you've considered**
If there is a good workaround in the logs I'd be happy to use that, but otherwise it involves going into reverse engineering the DLL exported functions when produced.

**Additional context**
N/A

**Comment by rkervella** [comment]
Looks like a regression, it should not be obfuscated. I'll look into it.

**Comment by rkervella** [comment]
What commit are you testing this on?

**Comment by gettheguru** [comment]
Perhaps it has been fixed since - it was 0.6.0, the early Release. I haven’t been able to test later yet because of the other issue, so it may have been resolved. Apologies if so?

**Comment by rkervella** [comment]
Yup that's been fixed since then, see https://github.com/BishopFox/sliver/blob/master/server/gobfuscate/symbols.go#L104

We might change that in the future, but so far the `RunSliver` export is not obfuscated anymore.
I'm working on fixing #172 so that you can test.
